stacks_queues/stack.py

```diff
@@ -21,18 +22,22 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.empty():
+            raise StackEmptyException("Stack is empty")
+            return None
+        else:
+            return self.store.remove_first()
```

Comment:
According to the specification, we should always return `None`, even when there is an element to remove. You could explicitly `return None` after this line, but a function will automatically return None if we don't return anything else
```suggestion
           self.store.remove_first()
```